frontend/src/components/pages/CreateReview/CreateReview.tsx

```diff
@@ -41,12 +58,15 @@ const CreateReview = (): React.ReactElement => {
   const [books, setBooks] = useState<Book[]>([]);
   const [review, setReview] = useState("");
   const [featured, setFeatured] = useState("0");
+  const [reviewerByline, setReviewerByline] = useState("");
 
   // const onBookModalClose = () => setShowBookModal(false);
```

Comment:
do we need this?

Reply:
I didn't touch these as part of my PR - they were all there from the previous PR by @uyxela.